wp-seo-main.php

```diff
@@ -624,3 +626,18 @@ function yoast_wpseo_self_deactivate() {
 		}
 	}
 }
+
+/**
+ * Excludes vendor directories from php-compatibility-checker.
+ *
+ * @since 9.4
+ *
+ * @param array $ignored Array of ignored directories/files.
+ *
+ * @return array Array of ignored directories/files.
+ */
+function yoast_wpseo_phpcompat_whitelist( $ignored ) {
+	$ignored[] = '*/wordpress-seo/vendor*';
```

Comment:
This needs to use the `plugin_dir_path` as there is no guarantee that the Plugin is installed in the `wordpress-seo` directory.

I would prefer to have the `vendor` and `vendor-prefixed` as separate entries for clarity.